### PR TITLE
fix: Add Zulu/Isizulu News Classification

### PR DESCRIPTION
**Pull request by isaac-chung**
## Checklist for adding MMTEB dataset

Reason for dataset addition: Cover [Zulu / isiZulu](https://en.wikipedia.org/wiki/Zulu_language) in Classification

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] I have considered the size of the dataset and reduced it if it is too big (2048 examples is typically large enough for most tasks)
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

File: mteb/tasks/Classification/zul/IsiZuluNewsClassification.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        description="isiZulu News Classification Dataset",
+        reference="https://huggingface.co/datasets/dsfsi/za-isizulu-siswati-news",
+        dataset={
+            "path": "isaacchung/isizulu-news",
+            "revision": "55caf0e52693a1ea63b15a4980a73fc137fb862b",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["train"],
+        eval_langs=["zul-Latn"],
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/zul/IsiZuluNewsClassification.py:21]: What about Siswati? (the code seems to be ssw)

**Review by KennethEnevoldsen — APPROVED** [review]
Looks good. Only a few comments

**Comment by imenelydiaker** [comment]
Just for my own culture: is Zulu and Swati the same language?

File: mteb/tasks/Classification/zul/IsiZuluNewsClassification.py (hunk 1 above)
Comment (isaac-chung) [reply mteb/tasks/Classification/zul/IsiZuluNewsClassification.py:21]: I only copied over the Zulu one for now. Will do the Siswati one separately. When loading the original dataset I encountered an error. :/

**Comment by isaac-chung** [comment]
> Just for my own culture: is Zulu and Swati the same language?

It seems like they have very close linguistic affiliation. Most comments say that they are close that it's easy to understand each other, but they are not the same. I was also reading https://en.wikipedia.org/wiki/Swazi_language

File: mteb/tasks/Classification/zul/IsiZuluNewsClassification.py (hunk 1 above)
Comment (KennethEnevoldsen) [reply mteb/tasks/Classification/zul/IsiZuluNewsClassification.py:21]: Ahh right. Should they be in the same dataset?

**Pull request merged**